src/imitation/algorithms/dagger.py

```diff
@@ -19,8 +19,9 @@
 from stable_baselines3.common.vec_env.base_vec_env import VecEnvStepReturn
 from torch.utils import data as th_data
 
+import imitation.data.serialize
 from imitation.algorithms import base, bc
-from imitation.data import rollout, types
+from imitation.data import rollout, serialize, types
```

Comment:
Why `import imitation.data.serialize` and also `from imitation.data import serialize`? Better to keep things consistent.

Reply:
Good point. Fixed it.